examples/iam-group-with-policies/main.tf

```diff
@@ -65,6 +65,35 @@ module "iam_group_with_custom_policies" {
   ]
 }
 
+#####################################################################################
+# IAM group to test the `create_group = false` option
+#####################################################################################
+module "iam_group_with_custom_policies_disabled" {
+  source = "../../modules/iam-group-with-policies"
+
+  create_group = false
+
+  name = "custom-disabled"
```

Comment:
When `create_group` is set to `false,` then it expects that the IAM group with the name specified in `name` has already been created. Also, this module adds provided properties (such as `group_users`, `custom_group_policy_arns`, ...) to it.

Reply:
@antonbabenko with due respect, but I don't see the same the behaviour being followed in the other modules, `iam-user` for example.

Reply:
If we had called this module `iam-group,` then the scope of resources should be - `all` or `nothing` (like in `iam-user`), but since it is `iam-group-with-policies`, it expects IAM group in one way or another.

Reply:
I found this behaviour pretty confusing @antonbabenko, but fair enough, thank you for the explanation 🙏

Reply:
I agree it is confusing.